src/main/scala/com/philcoutinho/gold/MediaCommentCount.scala

```diff
@@ -0,0 +1,12 @@
+package com.philcoutinho.gold
+
```

Comment:
remove redundant spaces... apply to entire project

Reply:
done